Modules/Core/ImageFunction/include/itkWindowedSincInterpolateImageFunction.h

```diff
@@ -343,14 +343,14 @@ class ITK_TEMPLATE_EXPORT WindowedSincInterpolateImageFunction : public Interpol
   TWindowFunction m_WindowFunction;
 
   /** Size of the offset table */
-  static constexpr unsigned int m_OffsetTableSize{ Math::UnsignedPower(m_WindowSize, ImageDimension) };
+  static constexpr SizeValueType m_OffsetTableSize{ Math::UnsignedPower(m_WindowSize, ImageDimension) };
```

Comment:
FYI, (If I recall correctly) VS2017 can also be made happy by using `=` instead of curly braces, as follows:

    static constexpr unsigned int m_OffsetTableSize = Math::UnsignedPower(m_WindowSize, ImageDimension);

Reply:
You are right, this is a better fix.